PayPal/build.gradle

```diff
@@ -32,6 +32,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation project(':BraintreeDataCollector')
 
+    implementation('com.paypal.checkout:android-sdk:0.1.0') {
+        exclude group: 'com.paypal.android.sdk'
```

Comment:
General knowledge: Is the base PayPal Android SDK not needed for native checkout?

Reply:
Braintree and the Checkout Android SDK have a dependency on magnes so this resolves the duplicate dependency, we could probably exclude on the artifact id as well though.